test/integration/coredump/test_coredump.py

```diff
@@ -0,0 +1,178 @@
+        if not os.path.isabs(core_wildcard):
+            core_wildcard = os.path.join(session_tmpdir, core_wildcard)
+    elif re.search(r"apport", core_pattern):
+        core_wildcard = os.path.join('/var/crash', '*.crash')
+        to_coredump = apport_crash_to_coredump
+    elif re.search(r"systemd-coredump", core_pattern):
+        core_wildcard = os.path.join('/var/lib/systemd/coredump', '*')
+    else:
+        assert False, "Unexpected core pattern '{}'".format(core_pattern)
+    cores = set(glob.glob(core_wildcard))
+
+    # Setup ulimit -c
+    rlim_core_soft, rlim_core_hard = resource.getrlimit(resource.RLIMIT_CORE)
+    if rlim_core_soft != resource.RLIM_INFINITY:
+        resource.setrlimit(resource.RLIMIT_CORE, (resource.RLIM_INFINITY, rlim_core_hard))
+    # Crash tarantool
+    cmd = ["tarantool", "-e", "require('ffi').cast('char *', 0)[0] = 42"]
+    rc, output = run_command_and_get_output(cmd, cwd=session_tmpdir)
+    # Restore ulimit -c
+    resource.setrlimit(resource.RLIMIT_CORE, (rlim_core_soft, rlim_core_hard))
+    assert rc != 0
+    assert re.search(r"Segmentation fault", output)
+
+    # Find the newly generated coredump
+    new_cores = set(glob.glob(core_wildcard)) - cores
+    assert len(new_cores) == 1
+
+    # And move it to the temporary directory (this directory is removed
+    # automatically, so there is no need to remove the coredump explicitly).
+    core_path = os.path.join(session_tmpdir, "core")
+    os.rename(next(iter(new_cores)), core_path)
+
+    assert os.path.exists(core_path)
+    return core_path if to_coredump is None else to_coredump(core_path)
+
+
+@pytest.fixture(scope="session")
+def coredump_packed(tt_cmd, coredump):
+    cmd = [tt_cmd, "coredump", "pack", coredump]
+    rc, _ = run_command_and_get_output(cmd, cwd=os.path.dirname(coredump))
+    assert rc == 0
+    files = glob.glob(os.path.join(os.path.dirname(coredump), '*.tar.gz'))
+    assert len(files) == 1
+    return files[0]
+
+
+@pytest.fixture(scope="session")
+def coredump_unpacked(tt_cmd, coredump_packed):
+    cmd = [tt_cmd, "coredump", "unpack", coredump_packed]
+    rc, _ = run_command_and_get_output(cmd, cwd=os.path.dirname(coredump_packed))
+    assert rc == 0
+    packed_name = os.path.basename(coredump_packed)
+    unpacked = os.path.join(os.path.dirname(coredump_packed), packed_name.split('.')[0])
+    assert os.path.isdir(unpacked)
+    return unpacked
+
+
+def test_coredump_pack_no_arg(tt_cmd, tmpdir):
+    cmd = [tt_cmd, "coredump", "pack"]
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc != 0
+    assert re.search(r"tt coredump pack", output)
+
+
+def test_coredump_pack_no_such_file(tt_cmd, tmpdir):
+    cmd = [tt_cmd, "coredump", "pack", "wrong_core_file"]
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc != 0
+    assert re.search(r"there was some problem packing archive.", output)
+
+
+@pytest.mark.skipif(os.getenv('ENABLE_COREDUMP_FIXTURE') is None,
```

Comment:
Maybe we should add TT_ prefix?:
TT_ENABLE_....

Reply:
Well... I doubt it. If we add TT_ prefix then why only here? Should we add similar prefix to other names as well? For example use tt_coredump fixture instead of coredump.

Reply:
Or it's because here we step out to the public scope and need to make some extra action to avoid ambiguity? Then it make sense.

Reply:
Done